GiEnJul/ClientApp/src/components/admin/overview/Recipient.tsx

```diff
@@ -2,17 +2,24 @@ import { Container, Grid, TextField } from "@material-ui/core";
 import { Search } from "@material-ui/icons";
 import * as React from "react";
 import { useEffect, useState } from "react";
-import DatatableRecipient from "../common/DataTableRecipient";
+import Datatable from "../common/RecipientTable";
 import { GiverType } from "./Giver";
 
+export interface PersonType {
```

Comment:
This is very similar to the PersonType in InstitutionForm.tsx. I think it the PersonType should be placed into common, so the same type can be used both places. I don't think that the partitionKey and rowKey is necessary. If they are you could extend the type in the common folder.